.github/workflows/loadtest.yaml

```diff
@@ -46,8 +46,9 @@ jobs:
     - name: Collect results
       working-directory: ./tests/load-tests
       env:
-        OCP_TOKEN: ${{ secrets.OCP_TOKEN }}
         STAGE_MEMBER_CLUSTER: ${{ secrets.STAGE_MEMBER_CLUSTER }}
+        PROMETHEUS_HOST: ${{ secrets.PROMETHEUS_HOST }}
```

Comment:
Hello. Would it be possible to get this dynamically in the test?

Reply:
I checked it, he Prometheus related SA token does not have authorization to query for the PROMETHEUS_HOST

Reply:
@naftalysh you don't need to read the route to get the full prometheus URL. You can actually construct it from the information you already have.

The Prometheus URL is something like `https://thanos-querier-openshift-monitoring.apps.${BASE_URL}>/api` where the `BASE_URL` is also part of the API URL (provided by the `STAGE_MEMBER_CLUSTER` variable, or you can get that by running `oc whoami --show-server`).

That API URL is something like `https://api.${BASE_URL}:6443`. You can extract the `BASE_URL` from it dynamically and then use it to construct the actual Prometheus URL

Reply:
@pmacik I've implemented the change and the load-test workflow succeeded